specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -481,12 +481,20 @@ def verify_cell_kzg_proof_batch_impl(row_commitments: Sequence[KZGCommitment],
                                      cosets_evals: Sequence[CosetEvals],
                                      proofs: Sequence[KZGProof]) -> bool:
     """
-    Verify a set of cells, given their corresponding proofs and their coordinates (row_index, column_index) in the blob
-    matrix. The i-th cell is in row row_indices[i] and in column column_indices[i].
-    The list of all commitments is provided in row_commitments_bytes.
+    Helper: Verify that a set of cells belong to their corresponding commitment.
 
-    This function is the internal implementation of verify_cell_kzg_proof_batch.
+    Specifically, given a set of tuples (``row_index``, ``column_index``, ``evals``, ``proof``), for every tuple it
```

Comment:
Nit: not sure if "a set of tuples" is really the input that this function gets. I understand what you mean, but maybe there is a better way to phrase it, e.g., "given a list of tuples (...) represented as four lists, ... ". Not sure.

Same comment applies for the public function.